fem/pgridfunc.hpp

```diff
@@ -434,13 +434,23 @@ class ParGridFunction : public GridFunction
    /// be used for ASCII output.
    void Save(const char *fname, int precision=16) const override;
 
-   /// Returns a GridFunction on MPI rank @a save_rank that does not have any
-   /// duplication of vertices/nodes at processor boundaries.
-   /// serial_mesh is obtained using ParMesh::GetSerialMesh(save_rank).
-   /// Note that the @ save_rank argument must match for the
-   /// ParMesh::GetSerialMesh and GetSerialGridFunction method.
+   /// @brief Returns a GridFunction on MPI rank @a save_rank that does not have
+   /// any duplication of vertices/nodes at processor boundaries.
+   ///
+   /// The @a serial_mesh is obtained using ParMesh::GetSerialMesh(save_rank).
+   /// Note that the @a save_rank must be the same as that used in
+   /// ParMesh::GetSerialMesh.
    GridFunction GetSerialGridFunction(int save_rank, Mesh &serial_mesh) const;
 
+   /// @brief Returns a GridFunction on MPI rank @a save_rank that does not have
+   /// any duplication of vertices/nodes at processor boundaries.
+   ///
+   /// The given @a serial_fes must be defined on the mesh returned by
+   /// ParMesh::GetSerialMesh(save_rank). Ownership of @a serial_fes is not
```

Comment:
It might be worth highlighting here that the `serial_fes` is typically obtained first time using the other method above (and that the obtained gridfunction owns the `serial_fes`).